client/state/user-devices/reducer.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	USER_DEVICES_ADD,
+} from 'state/action-types';
+
+export default createReducer( {}, {
+	[ USER_DEVICES_ADD ]: ( state, { devices } ) => devices
```

Comment:
well, maybe the API currently returns the full list of devices. what if we want to manually inject one? what if we want to hook up a web socket and inject a new device as soon as it gets added?

would be very unfortunate if adding a single device wiped out all the existing ones